src/test/skript/tests/syntaxes/expressions/ExprPortalCooldown.sk

```diff
@@ -0,0 +1,16 @@
+test "portal cooldown":
+	spawn zombie at spawn of world "world":
+		assert event-entity's portal cooldown is 0 seconds with "entity spawned with a portal cooldown"
+		set event-entity's portal cooldown to 25 ticks
+		assert event-entity's portal cooldown is 25 ticks with "portal cooldown set failed"
+		add 5 seconds to event-entity's portal cooldown
+		assert event-entity's portal cooldown is 125 ticks with "portal cooldown add ##1 failed"
+		remove 12 ticks from event-entity's portal cooldown
+		assert event-entity's portal cooldown is 113 ticks with "portal cooldown remove ##1 failed"
+		remove 999 ticks from event-entity's portal cooldown
+		assert event-entity's portal cooldown is 0 ticks with "portal cooldown remove ##2 failed"
+		delete event-entity's portal cooldown
+		assert event-entity's portal cooldown is 0 ticks with "portal cooldown delete failed"
+		reset event-entity's portal cooldown
+		assert event-entity's portal cooldown is 15 seconds with "portal cooldown reset failed"
+		delete event-entity
```

Comment:
Does this actually delete the entity? since you're deleting it before it's actually spawned

Reply:
I'll have to check, I just assumed it did since LimeGlass suggested it.

Reply:
yep, seems to work as intended